runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsPlugin.java

```diff
@@ -245,6 +245,11 @@ Color getGroupNameColor(final String name)
 		return null;
 	}
 
+	InventoryTagsConfig.DisplayMode getDisplayMode()
```

Comment:
Instead of creating a pseudo-getter here, inject `InventoryTagsConfig` into the overlay and call the config getter method directly there.